src/libsystemd-network/dhcp-server-internal.h

```diff
@@ -64,6 +64,7 @@ struct sd_dhcp_server {
         bool emit_router;
 
         Hashmap *leases_by_client_id;
+        Hashmap *static_leases_by_client_id;
```

Comment:
what is a static lease supposed to be? this is supposed to bind a specific MAC to a specific IP address?

Reply:
Yes, that is correct.